README.md

```diff
@@ -90,28 +91,28 @@ output "subscription_name" {
 }
 ```
 
-**Init, Plan and Apply**
+### 2. Init, Plan and Apply
 
 **Inputs**
 
+| Name                           | Description                                                  | Type      | Default                  |
+| ------------------------------ | ------------------------------------------------------------ | --------- | ------------------------ |
+| resource_prefix                | Prefix to be used for naming new resources                   | `string`  | `uptycs-integration-123` |
+| uptycs_app_client_id           | The Client ID of Uptycs Multi-tenant app                     | `string`  | required                 |
+| use_existing_service_principal | Whether to create a new service princial or use existing one | `boolean` | required                 |
 
-| Name            | Description                                | Type     | Default                             |
-| ----------------- | -------------------------------------------- | ---------- | ------------------------------------- |
-| resource_prefix | Prefix to be used for naming new resources | `string` | `uptycs-integration-123` |
-| uptycs_app_client_id | The Client ID of Uptycs Multi-tenant app | `string` | required |
-| use_existing_service_principal | Whether to create a new service princial or use existing one | `boolean` | required |
-
-**Outputs**
-
-
-| Name              | Description                         |
-| ------------------- | ------------------------------------- |
-| subscription_id   | Subscriptionid of the Azure Account |
-| subscription_name | Name of the the azure subscription  |
+**IMPORTANT NOTE:** If you have already integrated another subscription individually from the same Azure Tenant, then change the value of `use_existing_service_principal` to `true` in the above Terraform script before executing it. The Terraform will re-use the same Service Principal in that case.
```

Comment:
:::note Important
If you have already integrated another subscription individually from the same Azure Tenant, then change the value of `use_existing_service_principal` to `true` in the above Terraform script before executing it. The Terraform will re-use the same Service Principal in that case.
:::

Reply:
you can ignore since its not going in our stack